app/components/TabControl/PersistentTabControl.js

```diff
@@ -10,12 +10,12 @@ const Container = styled.div`
   display: ${props => (props.isActive ? 'display' : 'none')};
 `
 
-const PersistentTabControl = ({ activeTab, children }) => (
+const PersistentTabControl = ({ activeTab, children, className }) => (
```

Comment:
Where is the additional `className` prop used? I don't see it being passed in anywhere.

Reply:
https://github.com/LN-Zap/zap-desktop/blob/962cdb3eaf9f6806b5d24ae3f8cb01a8ea3317e0/app/components/TabControl/PersistentTabControl.js#L18

Here. Since tab control has not single parent (but a wrapping Fragment) className is applied to and individual tab, giving ability to alter styled of the wrapping div (the one that is used to hide/show tabs)